todo.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable eqeqeq */
+/* eslint-disable consistent-return */
+/* eslint-disable linebreak-style */
+/* eslint-disable array-callback-return */
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line linebreak-style
+let BookList = {};
+const DisplayBooks = document.querySelector('.collection');
+const NewTitle = document.querySelector('#title');
+const NewAuthor = document.querySelector('#author');
+const SaveBooks = localStorage.getItem('BookList');
+const AddBook = document.querySelector('#btn');
+let StoreBooks = [];
+const ClearFields = () => {
+  NewTitle.value = '';
+  NewAuthor.value = '';
+};
+const DisplayAllBooks = () => {
+  ClearFields();
+  DisplayBooks.innerHTML = '';
+  if (SaveBooks == null) {
+    [];
+  } else {
+    StoreBooks = JSON.parse(localStorage.getItem('BookList'));
+    StoreBooks.forEach((Books, index) => {
+      DisplayBooks.innerHTML += `
+        <div class="BookList">
+        <h1 class="titles">${Books.Title}</h1>
+        <h2 class="authors">${Books.Author}</h2>  
+        <button class="remove" id="${index}" onclick="RemoveBooks(event)">Remove</button>
+        <hr class="rule"> 
+        </div>`;
+    });
+  }
+};
+
+const SaveAllBooks = () => {
+  AddBook.addEventListener('click', () => {
+    BookList = {
+      Title: NewTitle.value,
+      Author: NewAuthor.value,
+    };
+    StoreBooks.push(BookList);
+    localStorage.setItem('BookList', JSON.stringify(StoreBooks));
+    DisplayAllBooks();
+    ClearFields();
+  });
+};
```

Comment:
- [ ] Your project is working :+1:  But when I enter a value, the data is not displayed instantly, I need to refresh the page. also If I click on the add book button. empty fields are saved and displayed. Please fix it in order to make your app work as expected :smile: 

![Screenshot from 2022-08-09 23-31-53](https://user-images.githubusercontent.com/6615394/183779898-620d0b2c-6852-40a0-b3b4-6303eda422f7.png)

Reply:
validation added to ensure no empty data is sent to local storage